lib/appliance_console.rb

```diff
@@ -505,6 +506,7 @@ module ApplianceConsole
           if database_configuration.activate
             database_configuration.post_activation
             say("\nConfiguration activated successfully.\n")
+            dbhost, dbtype, database = ApplianceConsole::Utilities.db_host_type_database
```

Comment:
Is this at the correct scope? If these local variables aren't defined above, because it's not yet configured, will defining them here be accessible when we start back at the beginning if the loop?